server/portal/settings/settings_custom.example.py

```diff
@@ -116,7 +116,8 @@
         'name': 'Shared Workspaces',
         'scheme': 'projects',
         'api': 'tapis',
-        'icon': None
+        'icon': None,
+        'read_only': False
```

Comment:
Perhaps we should add this to `settings_default.py` as well?

Reply:
👍🏼